### DISCUSSION TITLE
Expected release for Pinocchio 3.3 in PyPi

### DISCUSSION THREAD

**Discussion by jaydguthrie** [stack-of-tasks/pinocchio#2497] [General]
Pinocchio 3.3 is available in conda and homebrew. Is there an expected timeline for v3.3 to be available in PyPi (as of today it has version 2.7) ?
I briefly looked into cmeel to create my own wheel, but could not fully understand its usage.

**Comment by nim65s** [comment]
This is on my todolist for the end of the year :)

**Reply by jcarpent** [reply]
Thanks to @nim65s for the nice efforts to update Pinocchio on PyPi.
